### PR TITLE
Adding average stats

### PR DESCRIPTION
**Pull request by dominickendrick**
This PR adds change against weekly average for clickthrough and open rate stats.

**after:**
screen shot 2016-02-17 at 16 51 32

### CODE REVIEW

**Comment by dominickendrick** [comment]
@janua

File: app/models/EmailStatsSeriesData.scala [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      filterByDay(this.uniqueOpens),
+      filterByDay(this.numberSent),
+      filterByDay(this.numberDelivered),
+      filterByDay(this.unsubscribes)
+    )
+  }
+
+  def filterByDay(items: List[DateTimePoint]): List[DateTimePoint] = {
+    //work out total duration of time series
+    val start = new DateTime(items.head.timestamp)
```
Comment (janua) [inline app/models/EmailStatsSeriesData.scala:58]: Are you looking to frustrate yourself?

Comment (dominickendrick) [inline app/models/EmailStatsSeriesData.scala:58]: I hope that by remodelling the data, we can get rid of all this cr@p

**Comment by janua** [comment]
Looks good :+1: 
With the moving of code it's hard to spot the average code!

**Comment by dominickendrick** [comment]
Yeah, soz

**Pull request merged**